src/DynamoCoreWpf/ViewModels/Core/ConnectorPinViewModel.cs

```diff
@@ -51,7 +51,7 @@ public virtual void OnRequestRemove(Object sender, EventArgs e)
         #region Properties
 
         [JsonIgnore]
-        private readonly WorkspaceViewModel WorkspaceViewModel;
+        public readonly WorkspaceViewModel WorkspaceViewModel;
```

Comment:
does `internal` work?

Reply:
Changed to `private`. Unsure why the modifier was changed here.